test/e2e/apimachinery/aggregator.go

```diff
@@ -314,7 +314,23 @@ func TestSampleAPIServer(f *framework.Framework, image string) {
 	// is setting up a new namespace, we are just using that.
 	err = framework.WaitForDeploymentComplete(client, deployment)
 
-	// We seem to need to do additional waiting until the extension api service is actually up.
+	// Waiting until the extension api service is actually up.
+	err = wait.Poll(100*time.Millisecond, 30*time.Second, func() (bool, error) {
```

Comment:
I didn't expect an additional poll to be required here. Update the poll below to also retry on 503 errors